ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -180,3 +180,11 @@ def all(self, axis=None, keepdim=False, dtype=None, name=None):
     @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
     def sort(self, axis=-1, descending=False, name=None):
         return ivy.sort(self._ivy_array, axis=axis, descending=descending)
+
+    @with_supported_dtypes(
+        {"2.4.2 and below": ("bool", "uint8", "int8", "int16", "int32", "int64")},
+        "paddle",
+    )
+    def bitwise_xor(self, y, out=None, name=None):
+        y_ivy = _to_ivy_array(y)

```

Comment:
Is there is a specific reason why we use `_to_ivy_array`? I remember that we used `@to_ivy_arrays_and_back()` for this conversion but I can't find it in the rest of the file so I'm checking why it isn't there

Reply:
@KareemMAX  we are implementing method in Tensor class here and we take our first input using `self ` and by using `self `  where can convert our input into ivy array so no need of `@to_ivy_arrays_and_back()`. 
I am using `_to_ivy_array`   because if i use `@to_ivy_arrays_and_back()` then it will became redundant with this `self._ivy_array`

Reply:
Yup, I got mixed up here

But I'm still curious that in other functions it's calling `paddle_frontend` like in `tensor.multiply()` https://github.com/unifyai/ivy/pull/17416/files#diff-4b9f6f2107068e5624a6f5d66bae5dae58c467a34f8c5aad65df3478ce4a5abbR187

Reply:
```
@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
    def multiply(self, y, name=None):
        return paddle_frontend.multiply(self, y)
```

The implementation appears to be incorrect based on my understanding. According to my understanding, `paddle.tensor.Tensor `should receive functions from the backend, rather than the frontend. However, the above function is using `paddle.tensor.tensor.math.py `to access the backend.

Reply:
`paddle_frontend` is the ivy implementation of paddle frontend. We are not calling any paddle functions if that is what do you mean.

I've asked @hmahmood24 and he said the following:
>  since instance methods operate on the instance of the frontend class (`paddle.Tensor` in this case) which have a `_ivy_array` attribute that holds the `ivy.Array` already) and there are only 2 ways we go about writing these instance methods:
>
> Calling `ivy.<func>` directly
> Calling a frontend function of the same functionality `paddle_frontend.<func>`

That's why I think this implementation is wrong and `multiply()` is have the correct implementation

Reply:
@KareemMAX  done